app/templates/components/public/speaker-item.hbs

```diff
@@ -1,7 +1,7 @@
 {{#ui-accordion}}
   <div class="title">
     <div class="ui">
-      <img alt="speaker" class="ui medium rounded image" src="{{if speaker.thumbnailImageUrl speaker.thumbnailImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">
+      <img alt="speaker" class="ui rounded image speakerImg" src="{{if speaker.thumbnailImageUrl speaker.thumbnailImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">
```

Comment:
rounded is not being applied. @kushthedude Should speaker images be rounded? If yes, it is not working. If not, then remove rounded class. Scaling might have been working previously due to rounded class being applied

Reply:
The rounded class just means the corners won't be sharp, For actual round images, Semantic UI uses `circular` class.
Scaling issue is happening as we have recommended size of 300*300 px but when a speaker uploads an image of low pixels, to prevent pixelation and preserve the quality UI displays them in the original size. Scaling will only result in further pixelation and blurring of images.

Reply:
> Scaling issue is happening as we have recommended size of 300*300 px
If that is indeed true, then the cropper which appears when the user is uploading a picture should impose the square constraint -- like Instagram originally used to do. In that case, there should be no rectangular images. If

Reply:
@cosmiccoder96 There is no cropper modal for the uploading of speaker images. It is stored as it is.

Reply:
@CosmicCoder96 i didn't really get , where and what ti change, can you please explain a bit.

Reply:
@kushthedude Then add it, it will take care of it the right way, no amount of CSS will make a rectangular image look good if it forces it to a square.

Reply:
Cool !

Reply:
https://stackoverflow.com/questions/11552380/how-to-automatically-crop-and-center-an-image

Reply:
@iamareebjamal but faces of speakers might get cropped off the edges if there is no control over the positioning of the square, which will be problematic for sections like featured speakers.